CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   By default, StrimziPodSets are used for Kafka Connect and Kafka Mirror Maker 2.
   If needed, `StableConnectIdentites` can be disabled in the feature gates configuration in the Cluster Operator.
 * Support for the ppc64le platform
+* Refactor logic for Cruise Control cpu capacity settings
```

Comment:
I would add a section `Changes, deprecations and removals` as we have in previous releases and explain there what exactly changed. This is quite important as it might impact users.

Reply:
Added. Is it ok?